lib/statistics/distribution/normal.rb

```diff
@@ -79,5 +79,59 @@ def density_function(value)
         euler/Math.sqrt(2 * Math::PI)
       end
     end
+
+    # Standard Normal Inverse distribution:
+    # References:
+    # https://en.wikipedia.org/wiki/Inverse_distribution
+    # http://www.source-code.biz/snippets/vbasic/9.htm
+    class StandardNormalInverse < StandardNormal
```

Comment:
I think it's better to change the name to inverse standard normal:
```suggestion
    class InverseStandardNormal < StandardNormal
```